index.js

```diff
@@ -0,0 +1,123 @@
+const OBJ_OF_DIGIT = {
+  0: 'zero',
+  1: {
+    0: '',
+    1: 'one',
+    2: 'two',
+    3: 'three',
+    4: 'four',
+    5: 'five',
+    6: 'six',
+    7: 'seven',
+    8: 'eight',
+    9: 'nine',
+  },
+  11: {
+    0: '',
+    1: 'eleven',
+    2: 'twelve',
+    3: 'thirteen',
+    4: 'fourteen',
+    5: 'fifteen',
+    6: 'sixteen',
+    7: 'seventeen',
+    8: 'eighteen',
+    9: 'nineteen',
+  },
+  2: {
+    0: '',
+    1: 'ten',
+    2: 'twenty',
+    3: 'thirty',
+    4: 'fourty',
+    5: 'fifty',
+    6: 'sixty',
+    7: 'seventy',
+    8: 'eighty',
+    9: 'ninety',
+  },
+};
+
+const ORDINALS = {
+  0: '',
+  1: ' thousand',
+  2: ' million',
+};
+
+const HUNDREDS = ' hundred';
+
+function getValue(id) {
+  return document.getElementById(id).value;
+}
+
+function getArray(str) {
+  const result = [];
+  for (let i = str.length; i > 0; i = i - 3) {
+    const firstDigitOfThree = !!str[i - 3] ? str[i - 3] : '';
+    const secondDigitOfThree = !!str[i - 2] ? str[i - 2] : '';
+    const thirdDigitOfThree = !!str[i - 1] ? str[i - 1] : '';
+    result.push(`${firstDigitOfThree}${secondDigitOfThree}${thirdDigitOfThree}`);
```

Comment:
```suggestion
    result.push([3,2,1].reduce((acc, n) =>  acc += str[i-n] || '', acc))
```

Reply:
what is going on in my code? how we can improve `getArray` function even more using `RegExp` and `.match` method?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match

Reply:
Done first part of comment. I will try to impove getArray by .match metod. Thanks!